ci/deploy.yml

```diff
@@ -0,0 +1,67 @@
+#######################
+# resources
+#######################
+
+resources:
```

Comment:
![image](https://user-images.githubusercontent.com/691940/51179925-3946a380-18c7-11e9-8683-064ef3600ee2.png)

This reminds me of

```java
/**
  * gets the foo
  * @returns a foo of type Foo
  */
  Foo getFoo() {
    return this.foo
  }
```

Reply:
hahahah